apps/astarte_pairing/config/releases.exs

```diff
@@ -0,0 +1,23 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2020 Ispirata Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import Config
+
+broker_url = System.get_env("PAIRING_BROKER_URL", "mqtts://broker.beta.astarte.cloud:8883/")
```

Comment:
I don't get why you left that string here.

Reply:
We are configuring the url from the config file. The `default`-field is removed from `config.ex`. The value it is obtained either by setting it a system env var or by getting its default value.
@rbino What's your opinion?